src/test/java/org/folio/rest/impl/OkapiMockServer.java

```diff
@@ -168,6 +169,9 @@
   private static final String INSTANCE_ID_UNDERLYING_RECORD_WITH_CYRILLIC_DATA = "ebbb759a-dd08-4bf8-b3c3-3d75b2190c41";
   private static final String INSTANCE_ID_WITHOUT_SRS_RECORD = "3a6a47ab-597d-4abe-916d-e31c723426d3";
 
+  private static final String REPLACE_WITH_RECORDS = "replace_with_records";
+  private static final String REPLACE_TOTAL_COUNT = "replace_total_count";
```

Comment:
Could you please add something to describe the nature of constant, e.g. if it is a property then - REPLACE_WITH_RECORDS_PROPERTY(KEY), etc.

Reply:
Renamed